static/js/ui_init.js

```diff
@@ -462,6 +463,7 @@ export function initialize_everything() {
     const user_status_params = pop_fields("user_status");
     const i18n_params = pop_fields("language_list");
 
+    realm_logo.rerender();
```

Comment:
I think we can rename this function `realm_logo.rerender` now that this is used to render logo initially.

Reply:
Sounds good.